pkg/controllers/user/pipeline/remote/github/github.go

```diff
@@ -320,6 +320,9 @@ func convertRepos(repos []github.Repository) []v3.SourceCodeRepository {
 		if repo.Language != nil {
 			r.Spec.Language = *repo.Language
 		}
+		if repo.DefaultBranch != nil {
```

Comment:
Nit: is there a reason for running not nil check before assignment? Why not just assign?

Reply:
To avoid dereference from nil pointers. I can use Getters instead of the nil checking here